src/main/java/org/gitlab4j/api/models/Project.java

```diff
@@ -592,7 +593,69 @@ public Project withPrintingMergeRequestLinkEnabled(Boolean printingMergeRequestL
         return (this);
     }
 
+    public ProjectStatistics getStatistics() {
+        return statistics;
+    }
+
+    public void setStatistics(ProjectStatistics statistics) {
+        this.statistics = statistics;
+    }
+
     public static final boolean isValid(Project project) {
         return (project != null && project.getId() != null);
     }
+
+    @Override
```

Comment:
Would prefer to not override toString(), as it would be inconsistent with the other model classes.  If one would like to get a String representation of the instance you can call the following, which will return formatted JSON:

```
String json = org.gitlab4j.api.utils.JacksonJson.marshal(objectInstance);
```